docs/examples/identity/identity_pipeline.py

```diff
@@ -20,20 +20,25 @@
 # CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 import nvidia.dali as dali
-import argparse
 
 
-def main(filename):
-    pipe = dali.pipeline.Pipeline(batch_size=3, num_threads=1, device_id=0)
-    with pipe:
-        data = dali.fn.external_source(device="cpu", name="DALI_INPUT_0")
-        pipe.set_outputs(data)
+def _parse_args():
```

Comment:
It's not a module, so I don't see a reason to make this function private.
```suggestion
def parse_args():
```